src/pages/Product/components/ColorButton/ColorButton.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./ColorButton.scss";
+
+const ColorButton = (props) => {
+  return (
+    <button
+      className={`colorButton ${props.boxColor}`}
+      onClick={() => props.handleColorChange(props.boxColor)}
+    ></button>
```

Comment:
```suggestion
const ColorButton = ({boxColor, handleColorChange}) => {
  return (
    <button
      className={`colorButton ${boxColor}`}
      onClick={() => handleColorChange(boxColor)}
    ></button>
```

wrap up 시간에 배운 객체 구조분해 할당도 적용해보세요~